client/post-editor/media-modal/index.jsx

```diff
@@ -210,7 +210,7 @@ module.exports = React.createClass( {
 		this.setView( ModalViews.IMAGE_EDITOR );
 	},
 
-	onImageEditorDone( error, blob, imageEditorProps ) {
+	onImageEditorDone( error, blob, imageEditorProps, media ) {
```

Comment:
Good point. However, image editor accepts `media` prop which could include `ID` as well. We should probably add this media `ID` in `getImageEditorProps` only if it exists.